landoapi/cli.py

```diff
@@ -61,15 +61,18 @@ def init(init_s3):
 
     db.create_all()
     alembic.stamp("head")
+    logging.info("created lando db")
```

Comment:
Is this for debugging?

Reply:
I added this but it doesn't actually display anything - I think because lando's logging hasn't been initiated yet in this code path. I'll remove it.